### PR TITLE
fix(core/webidl): pad static methods

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: src/core/webidl.js [hunk 1]
```diff
@@ -672,13 +677,11 @@ function writeMethod(meth, max, indent) {
     });
   var params = paramObjs.join(", ");
   var len = idlType2Text(meth.idlType).length;
-  if (meth.static) len += 7;
   var specialProps = [
     "getter",
     "setter",
     "deleter",
-    "legacycaller",
-    "stringifier",
```
Comment (saschanaz) [inline src/core/webidl.js]: Shouldn't we keep `stringifier`? `legacycaller` is removed but `stringifier` is not.

Comment (marcoscaceres) [reply src/core/webidl.js]: Whoops!

Comment (marcoscaceres) [reply src/core/webidl.js]: yeah, will add a test!

**Comment by marcoscaceres** [comment]
now with correct spelling 💯

**Comment by marcoscaceres** [comment]
+ respec@20.0.4

**Pull request merged**